tests/conftest.py

```diff
@@ -26,16 +26,6 @@
 from openassetio.test.manager import harness
 
 
-@pytest.fixture(autouse=True)
-def bal_plugin_env(bal_base_dir, monkeypatch):
```

Comment:
[plan] Looks like this will break `integrations.yml` in the main OpenAssetIO repo, since we don't `pip install` BAL there.

Reply:
Yeah I was thinking when I noticed this issue, it's the opposite case than we discussed and planned for.  

Regardless, I think your plan is sensible for this particular case.